iidm/iidm-converter-api/src/main/java/com/powsybl/iidm/import_/ImportOptions.java

```diff
@@ -6,28 +6,71 @@
  */
 package com.powsybl.iidm.import_;
 
+import java.util.ArrayList;
+import java.util.List;
+
 /**
  * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
  * @author Miora Ralambotiana <miora.ralambotiana at rte-france.com>
  */
 public class ImportOptions {
 
-    private boolean throwExceptionIfExtensionNotFound;
+    private boolean throwExceptionIfExtensionNotFound = false;
+
+    private boolean importFromBaseAndExtensionsFiles = false;
+
+    private boolean importFromBaseAndMultipleExtensionFiles = false;
+
+    private List<String> extensions = new ArrayList<>();
```

Comment:
- `ImportOptions` and `ExportOptions` must be consistent. Put `ALL` by default in both.
- You never really check if an extension is in this `List`. Did you miss something ?
- Use a `Set` rather than a `List`.